### PR TITLE
feat(perf-issues): Add basic UI for N+1 API Calls issue details page

### PR DESCRIPTION
**Pull request by gggritso**
Just a few small things so this page is functional for LA.

- Add "Alpha" tag to header
- Add new group type
- Improve span evidence annotation
- Add problem parameter section to evidence

**e.g.,**

Screen Shot 2023-01-05 at 2 28 36 PM

### CODE REVIEW

File: static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.tsx [hunk 1]
```diff
@@ -36,10 +36,20 @@ export function SpanEvidenceKeyValueList({
...
           ? t('Repeating Span')
           : t('Offending Span'),
-      value: getSpanEvidenceValue(offendingSpan),
+      value: getSpanEvidenceValue(offendingSpans[0]),
     },
   ];
 
+  const problemParameters = getProblemParameters(offendingSpans);
+
+  if (problemParameters && issueType === IssueType.PERFORMANCE_N_PLUS_ONE_API_CALLS) {
```
Comment (malwilley) [inline static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.tsx]: FYI (you may already be aware) this logic will need to be on the backend soon once the issue platform team pushes out occurrences: https://github.com/getsentry/sentry/pull/42366

**Review by malwilley — APPROVED** [review]

File: static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.tsx [hunk 2]
```diff
@@ -36,10 +36,20 @@ export function SpanEvidenceKeyValueList({
...
+    data.push({
+      key: '3',
+      subject: t('Problem Parameter'),
+      value: getProblemParameters(offendingSpans),
```
[the remark this answers is not in the document]
Comment (gggritso) [reply static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.tsx]: Yes it is, thanks! Removed

File: static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.tsx (hunk 1 above)
Comment (gggritso) [reply static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.tsx]: Whoa, I had no idea this was happening, thanks for the heads-up! Please let the Performance team know when this work gets merged, we have a bunch of new issue types in the hopper 🙏🏻

**Review by 0Calories — APPROVED** [review]
Solid! 

Maybe we could add a new test to `spanEvidence.spec.tsx` to get some coverage on the visuals for this new issue type?

**Comment by gggritso** [comment]
@0Calories you and Codecov agreed on that one. I moved some tests around and expanded the test for `SpanEvidenceKeyValueList` to check for the headings and values

**Review by 0Calories — APPROVED** [review]
Tests look good! ✅

**Pull request merged**